src/main/java/teammates/common/util/Const.java

```diff
@@ -50,7 +50,7 @@
     
     public static final String USER_NAME_FOR_SELF = "Myself";
     public static final String USER_TEAM_FOR_INSTRUCTOR = "Instructors";
-    public static final String NO_SPECIFIC_RECIEPIENT = "No specific recipient";
+    public static final String NO_SPECIFIC_RECIPIENT= "No specific recipient";
```

Comment:
The removed space before `=` causes red "X" in the build. Can you add it back again?